src/commands/dev/dev.js

```diff
@@ -325,8 +325,10 @@ const dev = async (options, command) => {
       '--trafficMesh and -t are deprecated and will be removed in the near future. Please use --edgeHandlers or -e instead.',
     )
   }
-
-  await injectEnvVariables({ env: command.netlify.cachedConfig.env, site })
+  if (options.env) {
+    devConfig.envPriority = options.env.split(' ')
```

Comment:
If we do keep the flag (see my other comment on scope), I suggest using a comma separated list, you can see and example in https://github.com/tj/commander.js/tree/02a124c7d58dbae2ef11f9284b2c68ad94f6dc8b#custom-option-processing.

WDYT?

Reply:
I can try with the comma separated list

Reply:
Done